storybookjs/storybook
# UI: Add remount story shortcut

Closes #18669

## What I did

- Added shortcut for remount tool. Default is `alt + R`
- Added remount tool shortcut to shortcut key bind menu.
  <img width="635" alt="Screen Shot 2023-03-05 at 2 02 30 AM" src="https://user-images.githubusercontent.com/18149746/222949168-4067e6cb-2848-43c1-8263-a7e4c78a2161.png">
- Added remount tool shortcut tests. 



## How to test

Click `alt + R` and verify remount button animates and works.
Set new key bind to remount shortcut. 
Reset key binds and verify it goes back to default.



## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
